src/components/manager.ts

```diff
@@ -190,14 +190,25 @@ export class Manager {
         }
     }
 
+    /**
+     * Returns the jobname. If empty, return the name of the input `texPath`.
+     *
+     * @param texPath The path of a LaTeX file.
+     */
+    jobname(texPath: string) {
+        const config = vscode.workspace.getConfiguration('latex-workshop')
```

Comment:
To access a configuration setting with scope `resource`, we need to pass a `ConfigurationScope` to `getConfiguration` like

```ts
vscode.workspace.getConfiguration('latex-workshop', vscode.Uri.file(texPath))
```

Reply:
Done!